frontend/src/components/navigation/UserMeta.vue

```diff
@@ -51,6 +65,12 @@ import { mapGetters } from 'vuex'
 export default {
   name: 'UserMeta',
   components: { UserAvatar },
+  props: {
+    justAvatar: {
```

Comment:
Obviously matter of taste, but I'd have rather named it `avatarOnly` (like readonly, writeonly, iconOnly, etc.)

The reason you had to introduce this, was because we haven't used this for mobile so far (see other comment)